scene/main/node.h

```diff
@@ -501,6 +501,13 @@ class Node : public Object {
 	Node *get_parent() const;
 	Node *find_parent(const String &p_pattern) const;
 
+	// Fast unsafe access to children, for bottleneck code.
+	// Use get_child_count() / get_child() for safe access.
+	// These are raw and include internal children.
+	// Callers *must* call `update_cached_children()` if there is a possibility cached children are not up to date.
+	void update_cached_children() const { _update_children_cache(); }
```

Comment:
Since this just delegates, it might make sense to change the name of the old method, to not have two names for the same method.

Reply:
Yup, absolutely have seen this done in previous PRs, however it would be nice to check first with a dev build there are no regressions, as otherwise this could create git noise if we wanted to revert for any reason. It's easy enough to change in a followup.